src/main/java/org/apache/commons/io/FileSystemUtils.java

```diff
@@ -293,7 +293,13 @@ long freeSpaceOS(final String path, final int os, final boolean kb, final long t
      * @return the amount of free drive space on the drive
      * @throws IOException if an error occurs
      */
+    @SuppressWarnings("nullness:assignment.type.incompatible") 
     long freeSpaceWindows(String path, final long timeout) throws IOException {
+        // normalize(String filename,boolean) returns null if its arugment is an invalid path.
+        // BUG : A simple check to verify(confirm) that path
+        // is legal/valid as per Windows system will resolve this bug.
+        // It seems the only values that are ever passed are "" and "/".
+        // Are they legal?  Does "/" flow to here?
```

Comment:
@mernst I have a test case, in which NPE occurs due to this bug. I will report it on apache issue tracking, Should I use "regex" to verify validity of Windows path ?